### PR TITLE
Simplified training json

### PR DESCRIPTION
**Pull request by tihsu99**

### CODE REVIEW

File: data/sample_2018UL.json [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-  "WpWpJJ_EWK" : [0.02597, 1, "VBS",   700,  1, 1],
-  "ZZJJTo4L"   : [0.008645, 1, "VBS",  700,  1, 1],
-  "WpWpJJ_QCD" : [0.02221, 1, "VBS",   700,  1, 1],
-  "EGamma":    [1,     0, "data",      0,   -1, 0],
-  "SingleMuon":[1,     0, "data",      0,   -1, 0],
-  "DoubleMuon":[1,     0, "data",      0,   -1, 0],
-  "MuonEG":    [1,     0, "data",      0,   -1, 0]
+  "DYnlo":   [6077.22,  1, "DY",        632,  0],
+  "WJets":   [61526.7,  1, "WJet",      593, -1],
+  "osWW":    [11.09,    1, "VV",        425,  0],
```
Comment (efeyazgan) [inline data/sample_2018UL.json:4]: So, no opposite sign WW anywhere?

**Comment by efeyazgan** [comment]
It would be good if you could explain the json structure in a readme.

**Comment by gourangakole** [comment]
Hi @AlbertHsuNTUphys , can you please do the changes to application condor file as well?

File: data/sample_2018UL.json (hunk 1 above)
Comment (tihsu99) [reply data/sample_2018UL.json:4]: Not in the training in original code.

**Comment by gourangakole** [comment]
Thank you @AlbertHsuNTUphys. as I discussed with you. 
this is just make the json simplification so just removed extra columns. But Terry with will make another PR with the latest bkgs composition and (correct weight as discussed in yesterday's meeting)
fyi @efeyazgan 

so I think its ready to merge.

**Pull request merged**